release_notes.md

```diff
@@ -13,3 +13,4 @@
     - Customers can opt-out of this behavior by setting `SendCanceledInvocationsToWorker` to `false` in host.json
   - If a worker does not support CancellationTokens, cancelled invocations will not be sent to the worker
 - Warn when `FUNCTIONS_WORKER_RUNTIME` is not set (#9799)
+- Add an app setting to allow CORS configuration (#9846)
```

Comment:
Wonder we should make this more specific. something like "Expanding CORS support for non Linux scenarios" or something along that line.

Reply:
The app setting would apply to all OS though. So it would be little misleading to say that.

Reply:
"Expanding" implies that this feature already exists for linux